### DISCUSSION TITLE
ModuleNotFoundError: No module named 'tomli'

### DISCUSSION THREAD

**Discussion by EJ-Stokvis** [asreview/asreview#1572] [Troubleshooting (report errors in the "Issues" tracker)]
Hi can someone help me as i have just updated asreview and get this error
<img width="561" alt="Screenshot 2023-11-07 at 14 14 40" src="https://github.com/asreview/asreview/assets/80335974/7f8863f5-c5b8-4c7b-83cd-6c7cb0f6add9">

**Comment by J535D165** [comment]
Can you run `pip install tomli`?
